src/Catalyst.Core.Modules.Keystore/LocalKeyStore.cs

```diff
@@ -125,16 +122,18 @@ private byte[] KeyStoreDecrypt(PasswordRegistryTypes passwordIdentifier, string
             throw new AuthenticationException("Password incorrect for keystore.");
         }
 
-        public async Task<IPrivateKey> KeyStoreGenerate(KeyRegistryTypes keyIdentifier)
+        public async Task<IPrivateKey> KeyStoreGenerate(NetworkType networkType, KeyRegistryTypes keyIdentifier)
         {
             var privateKey = _cryptoContext.GeneratePrivateKey();
 
-            await KeyStoreEncryptAsync(privateKey, keyIdentifier);
+            await KeyStoreEncryptAsync(privateKey, networkType, keyIdentifier);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4338061)